indy_node/server/domain_req_handler.py

```diff
@@ -305,6 +304,18 @@ def _validate_revoc_reg_entry(self, req: Request):
             revoc_reg_def_id=req.operation[REVOC_REG_DEF_ID],
             req_id=req.reqId
         )
+
+        is_owner = False
+
+        if get_req_id(req) == req.operation[REVOC_REG_DEF_ID]:
```

Comment:
This comparison is incorrect. Here Request ID is compared to REVOC_REG_DEF_ID which is RevocReg's key in State Trie.
But we need to compare sender's DID (`req.identifier`) with `revoc_def `'s DID (either get it from `revoc_def` or from `revoc_reg_def_id`, as this DID is part of `revoc_reg_def_id`, see `prepare_revoc_def_for_state` method).